ktor-server/ktor-server-core/api/ktor-server-core.api

```diff
@@ -985,8 +985,6 @@ public abstract interface class io/ktor/response/ApplicationResponse {
 
 public final class io/ktor/response/ApplicationResponseFunctionsKt {
 	public static final fun defaultTextContentType (Lio/ktor/application/ApplicationCall;Lio/ktor/http/ContentType;)Lio/ktor/http/ContentType;
-	public static final fun respond (Lio/ktor/application/ApplicationCall;Lio/ktor/http/HttpStatusCode;Ljava/lang/Object;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
```

Comment:
I would keep old functions a well, at least for binary compatibility

Reply:
sure, it's still a draft PR that needs more fine-tuning. I published it to just run tests on CI

Reply:
updated